### PR TITLE
chore(release): publish 9.1.0

### PR DESCRIPTION
**Pull request by michaelfig**
- @agoric/sdk@9.1.0
 - @agoric/cosmos@0.26.19
 - @agoric/ertp@0.12.1
 - @agoric/swingset-vat@0.22.1
 - @agoric/access-token@0.4.14
 - agoric@0.13.18
 - @agoric/assert@0.3.13
 - @agoric/bundle-source@1.4.11
 - @agoric/captp@1.10.5
 - @agoric/cosmic-swingset@0.34.1
 - @agoric/dapp-svelte-wallet@0.11.4
 - @agoric/deploy-script-support@0.5.4
 - @agoric/deployment@1.29.0
 - @agoric/eslint-config@0.3.20
 - @agoric/eventual-send@0.13.29
 - @agoric/governance@0.2.3
 - @agoric/import-bundle@0.2.30
 - @agoric/import-manager@0.2.30
 - @agoric/install-metering-and-ses@0.2.27
 - @agoric/install-ses@0.5.27
 - @agoric/marshal@0.4.27
 - @agoric/notifier@0.3.30
 - @agoric/pegasus@0.4.4
 - @agoric/promise-kit@0.2.27
 - @agoric/same-structure@0.1.27
 - @agoric/sharing-service@0.1.30
 - @agoric/solo@0.5.1
 - @agoric/sparse-ints@0.1.23
 - @agoric/spawner@0.4.30
 - @agoric/stat-logger@0.4.23
 - @agoric/store@0.6.5
 - @agoric/swing-store@0.6.1
 - @agoric/swingset-runner@0.17.1
 - @agoric/tame-metering@2.0.13
 - @agoric/transform-metering@1.4.26
 - @agoric/treasury@0.6.4
 - @agoric/ui-components@0.2.25
 - @agoric/vats@0.4.1
 - @agoric/xsnap@0.9.1
 - @agoric/zoe@0.19.1
 - @agoric/dapp-svelte-wallet-api@0.10.4
 - @agoric/dapp-svelte-wallet-ui@1.10.0

## Packages that have NEWS.md updates

```diff
--- a/packages/SwingSet/CHANGELOG.md
+++ b/packages/SwingSet/CHANGELOG.md
@@ -3,6 +3,26 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.22.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/swingset-vat@0.22.0...@agoric/swingset-vat@0.22.1) (2021-09-23)
+
+
+### Features
+
+* **TimerService:** add new `delay` method and protect device args ([7a2c830](https://github.com/Agoric/agoric-sdk/commit/7a2c830b6cdea1e81cc0eb8fef517704dc30a922))
+
+
+### Bug Fixes
+
+* **solo:** make `localTimerService` in ms, and update correctly ([d6d4724](https://github.com/Agoric/agoric-sdk/commit/d6d472445a05b8c3d83fc9621879c3c91bf4d737))
+* **swingset:** DummySlogger needs to be more realistic ([d8d146d](https://github.com/Agoric/agoric-sdk/commit/d8d146de69a08d2b6582b3a778a7f44985450fc0))
+* **swingset:** have slogger record replay status for syscalls too ([722f903](https://github.com/Agoric/agoric-sdk/commit/722f903c8306f4245addad6bbe9df7b8b7a7321b))
+* **swingset:** make vat-warehouse responsible for slogging deliveries ([e317589](https://github.com/Agoric/agoric-sdk/commit/e31758914c165113c4ba6a4574a73ee888addad1))
+* **SwingSet:** improve slogging during replay ([d6e64da](https://github.com/Agoric/agoric-sdk/commit/d6e64daf4d8240ad51eafb917e6f4590c8caebfb)), closes [#3428](https://github.com/Agoric/agoric-sdk/issues/3428)
+* **timer:** remove deprecated `createRepeater` ([b45c66d](https://github.com/Agoric/agoric-sdk/commit/b45c66d6d5aadcd91bd2e50d31104bce8d4d78f6))
+* **xsnap:** format objects nicely in console using SES assert.quote ([#3856](https://github.com/Agoric/agoric-sdk/issues/3856)) ([a3306d0](https://github.com/Agoric/agoric-sdk/commit/a3306d01d8e87c4bc7483a61e42cc30b006feb81)), closes [#3844](https://github.com/Agoric/agoric-sdk/issues/3844)
+
+
+
 ## [0.22.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/swingset-vat@0.21.3...@agoric/swingset-vat@0.22.0) (2021-09-15)
 
 
--- a/packages/agoric-cli/CHANGELOG.md
+++ b/packages/agoric-cli/CHANGELOG.md
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.13.18](https://github.com/Agoric/agoric-sdk/compare/agoric@0.13.17...agoric@0.13.18) (2021-09-23)
+
+
+### Features
+
+* **solo:** make client objects appear earlier, parallelise chain ([656514e](https://github.com/Agoric/agoric-sdk/commit/656514e5937389c57e139bc1302fa435edd2e674))
+
+
+
 ### [0.13.17](https://github.com/Agoric/agoric-sdk/compare/agoric@0.13.16...agoric@0.13.17) (2021-09-15)
 
 
--- a/packages/cosmic-swingset/CHANGELOG.md
+++ b/packages/cosmic-swingset/CHANGELOG.md
@@ -3,6 +3,20 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.34.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/cosmic-swingset@0.34.0...@agoric/cosmic-swingset@0.34.1) (2021-09-23)
+
+
+### Features
+
+* **solo:** make client objects appear earlier, parallelise chain ([656514e](https://github.com/Agoric/agoric-sdk/commit/656514e5937389c57e139bc1302fa435edd2e674))
+
+
+### Bug Fixes
+
+* **sim-chain:** update `chainTimerService` correctly ([3f49a77](https://github.com/Agoric/agoric-sdk/commit/3f49a779f253ff01fe7e71d0295efbfa99b669a9))
+
+
+
 ## [0.34.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/cosmic-swingset@0.33.5...@agoric/cosmic-swingset@0.34.0) (2021-09-15)
 
 
--- a/packages/dapp-svelte-wallet/CHANGELOG.md
+++ b/packages/dapp-svelte-wallet/CHANGELOG.md
@@ -3,6 +3,18 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.11.4](https://github.com/Agoric/agoric-sdk/compare/@agoric/dapp-svelte-wallet@0.11.3...@agoric/dapp-svelte-wallet@0.11.4) (2021-09-23)
+
+
+### Features
+
+* **wallet:** Add dismiss option for dapp connections ([#3836](https://github.com/Agoric/agoric-sdk/issues/3836)) ([f193fa2](https://github.com/Agoric/agoric-sdk/commit/f193fa225c6e2e88b067d43e22b1fcc1230ebb98))
+* **wallet:** Don't show autodeposit payments in requests, smooth out rendering animations ([#3832](https://github.com/Agoric/agoric-sdk/issues/3832)) ([b03c7a5](https://github.com/Agoric/agoric-sdk/commit/b03c7a5d44093c356d29e807a93041a760b1a029))
+* **wallet:** Eagerly show pending state for approved offers ([#3846](https://github.com/Agoric/agoric-sdk/issues/3846)) ([20abcbe](https://github.com/Agoric/agoric-sdk/commit/20abcbed5bfab6c5a8ad19462b9b726fd2d6490b))
+* **wallet:** Fix icons and list rendering bugs ([#3827](https://github.com/Agoric/agoric-sdk/issues/3827)) ([02b7554](https://github.com/Agoric/agoric-sdk/commit/02b7554076bddbe878756ac44af736c4b5d282c2))
+
+
+
 ### [0.11.3](https://github.com/Agoric/agoric-sdk/compare/@agoric/dapp-svelte-wallet@0.11.2...@agoric/dapp-svelte-wallet@0.11.3) (2021-09-15)
 
 
--- a/packages/dapp-svelte-wallet/ui/CHANGELOG.md
+++ b/packages/dapp-svelte-wallet/ui/CHANGELOG.md
@@ -3,6 +3,18 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.10.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/dapp-svelte-wallet-ui@1.9.0...@agoric/dapp-svelte-wallet-ui@1.10.0) (2021-09-23)
+
+
+### Features
+
+* **wallet:** Add dismiss option for dapp connections ([#3836](https://github.com/Agoric/agoric-sdk/issues/3836)) ([f193fa2](https://github.com/Agoric/agoric-sdk/commit/f193fa225c6e2e88b067d43e22b1fcc1230ebb98))
+* **wallet:** Don't show autodeposit payments in requests, smooth out rendering animations ([#3832](https://github.com/Agoric/agoric-sdk/issues/3832)) ([b03c7a5](https://github.com/Agoric/agoric-sdk/commit/b03c7a5d44093c356d29e807a93041a760b1a029))
+* **wallet:** Eagerly show pending state for approved offers ([#3846](https://github.com/Agoric/agoric-sdk/issues/3846)) ([20abcbe](https://github.com/Agoric/agoric-sdk/commit/20abcbed5bfab6c5a8ad19462b9b726fd2d6490b))
+* **wallet:** Fix icons and list rendering bugs ([#3827](https://github.com/Agoric/agoric-sdk/issues/3827)) ([02b7554](https://github.com/Agoric/agoric-sdk/commit/02b7554076bddbe878756ac44af736c4b5d282c2))
+
+
+
 ## [1.9.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/dapp-svelte-wallet-ui@1.8.2...@agoric/dapp-svelte-wallet-ui@1.9.0) (2021-09-15)
 
 
--- a/packages/deployment/CHANGELOG.md
+++ b/packages/deployment/CHANGELOG.md
@@ -3,6 +3,20 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.29.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/deployment@1.28.14...@agoric/deployment@1.29.0) (2021-09-23)
+
+
+### Features
+
+* **deployment:** use latest faucet-helper.sh from testnet ([83f45f6](https://github.com/Agoric/agoric-sdk/commit/83f45f6be8112b9e74f687d9436963051d9e5308))
+
+
+### Bug Fixes
+
+* **deployment:** wait for staking tokens before creating validator ([59952d3](https://github.com/Agoric/agoric-sdk/commit/59952d3afd78ceb9b00eac5a8ccc84bef9d2ee4b))
+
+
+
 ### [1.28.14](https://github.com/Agoric/agoric-sdk/compare/@agoric/deployment@1.28.13...@agoric/deployment@1.28.14) (2021-09-15)
 
 **Note:** Version bump only for package @agoric/deployment
--- a/packages/solo/CHANGELOG.md
+++ b/packages/solo/CHANGELOG.md
@@ -3,6 +3,23 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.5.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/solo@0.5.0...@agoric/solo@0.5.1) (2021-09-23)
+
+
+### Features
+
+* **solo:** make client objects appear earlier, parallelise chain ([656514e](https://github.com/Agoric/agoric-sdk/commit/656514e5937389c57e139bc1302fa435edd2e674))
+
+
+### Bug Fixes
+
+* **solo:** chain events come from both 'NewBlockHeader' and 'Tx' ([3cefa07](https://github.com/Agoric/agoric-sdk/commit/3cefa0758ab3c0397d7430d0b79b0e544f0d4249))
+* **solo:** don't enforce origin identity; we have access tokens ([3015ecd](https://github.com/Agoric/agoric-sdk/commit/3015ecdd71cbc5d811c2bd88e2d51096919e4754))
+* **solo:** make `localTimerService` in ms, and update correctly ([d6d4724](https://github.com/Agoric/agoric-sdk/commit/d6d472445a05b8c3d83fc9621879c3c91bf4d737))
+* **timer:** remove deprecated `createRepeater` ([b45c66d](https://github.com/Agoric/agoric-sdk/commit/b45c66d6d5aadcd91bd2e50d31104bce8d4d78f6))
+
+
+
 ## [0.5.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/solo@0.4.6...@agoric/solo@0.5.0) (2021-09-15)
 
 
--- a/packages/vats/CHANGELOG.md
+++ b/packages/vats/CHANGELOG.md
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.4.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/vats@0.4.0...@agoric/vats@0.4.1) (2021-09-23)
+
+
+### Features
+
+* **solo:** make client objects appear earlier, parallelise chain ([656514e](https://github.com/Agoric/agoric-sdk/commit/656514e5937389c57e139bc1302fa435edd2e674))
+
+
+
 ## [0.4.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/vats@0.3.2...@agoric/vats@0.4.0) (2021-09-15)
 
 
--- a/packages/xsnap/CHANGELOG.md
+++ b/packages/xsnap/CHANGELOG.md
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.9.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/xsnap@0.9.0...@agoric/xsnap@0.9.1) (2021-09-23)
+
+
+### Bug Fixes
+
+* **xsnap:** format objects nicely in console using SES assert.quote ([#3856](https://github.com/Agoric/agoric-sdk/issues/3856)) ([a3306d0](https://github.com/Agoric/agoric-sdk/commit/a3306d01d8e87c4bc7483a61e42cc30b006feb81)), closes [#3844](https://github.com/Agoric/agoric-sdk/issues/3844)
+
+
+
 ## [0.9.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/xsnap@0.8.2...@agoric/xsnap@0.9.0) (2021-09-15)
 
 
--- a/packages/zoe/CHANGELOG.md
+++ b/packages/zoe/CHANGELOG.md
@@ -3,6 +3,22 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.19.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/zoe@0.19.0...@agoric/zoe@0.19.1) (2021-09-23)
+
+
+### Features
+
+* **TimerService:** add new `delay` method and protect device args ([7a2c830](https://github.com/Agoric/agoric-sdk/commit/7a2c830b6cdea1e81cc0eb8fef517704dc30a922))
+
+
+### Bug Fixes
+
+* fix bug which breaks rights conservation and offer safety guarantees ([#3858](https://github.com/Agoric/agoric-sdk/issues/3858)) ([b67bfcb](https://github.com/Agoric/agoric-sdk/commit/b67bfcb9051cdcf780aff1a10653635448b21eae))
+* **timer:** remove deprecated `createRepeater` ([b45c66d](https://github.com/Agoric/agoric-sdk/commit/b45c66d6d5aadcd91bd2e50d31104bce8d4d78f6))
+* skip refill meter test ([#3849](https://github.com/Agoric/agoric-sdk/issues/3849)) ([90a456f](https://github.com/Agoric/agoric-sdk/commit/90a456f78f918ad01924da4b131f5a272d03624b))
+
+
+
 ## [0.19.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/zoe@0.18.1...@agoric/zoe@0.19.0) (2021-09-15)
 
 
```

### CODE REVIEW

File: packages/SwingSet/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,26 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [0.22.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/swingset-vat@0.22.0...@agoric/swingset-vat@0.22.1) (2021-09-23)
+
+
+### Features
+
+* **TimerService:** add new `delay` method and protect device args ([7a2c830](https://github.com/Agoric/agoric-sdk/commit/7a2c830b6cdea1e81cc0eb8fef517704dc30a922))
```
Comment (dckc) [inline packages/SwingSet/CHANGELOG.md:11]: The "protect device args" part should have been a `fix`.

File: packages/SwingSet/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,26 @@
...
+
+
+### Bug Fixes
+
+* **solo:** make `localTimerService` in ms, and update correctly ([d6d4724](https://github.com/Agoric/agoric-sdk/commit/d6d472445a05b8c3d83fc9621879c3c91bf4d737))
+* **swingset:** DummySlogger needs to be more realistic ([d8d146d](https://github.com/Agoric/agoric-sdk/commit/d8d146de69a08d2b6582b3a778a7f44985450fc0))
+* **swingset:** have slogger record replay status for syscalls too ([722f903](https://github.com/Agoric/agoric-sdk/commit/722f903c8306f4245addad6bbe9df7b8b7a7321b))
+* **swingset:** make vat-warehouse responsible for slogging deliveries ([e317589](https://github.com/Agoric/agoric-sdk/commit/e31758914c165113c4ba6a4574a73ee888addad1))
+* **SwingSet:** improve slogging during replay ([d6e64da](https://github.com/Agoric/agoric-sdk/commit/d6e64daf4d8240ad51eafb917e6f4590c8caebfb)), closes [#3428](https://github.com/Agoric/agoric-sdk/issues/3428)
```
Comment (dckc) [inline packages/SwingSet/CHANGELOG.md:20]: improved slogging seems like a `feat`

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,38 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [9.1.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/sdk@9.0.0...@agoric/sdk@9.1.0) (2021-09-23)
+
+
+### Features
+
+* **deployment:** use latest faucet-helper.sh from testnet ([83f45f6](https://github.com/Agoric/agoric-sdk/commit/83f45f6be8112b9e74f687d9436963051d9e5308))
```
Comment (dckc) [inline CHANGELOG.md:11]: ideally "use latest faucet-helper" would be rephrased to say what the new end-user feature is.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,38 @@
...
+* **solo:** make client objects appear earlier, parallelise chain ([656514e](https://github.com/Agoric/agoric-sdk/commit/656514e5937389c57e139bc1302fa435edd2e674))
+* **TimerService:** add new `delay` method and protect device args ([7a2c830](https://github.com/Agoric/agoric-sdk/commit/7a2c830b6cdea1e81cc0eb8fef517704dc30a922))
```
Comment (dckc) [inline CHANGELOG.md:13]: The "protect device args" part should have been a bug fix. (also noted elsewhere)

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,38 @@
...
+* **wallet:** Add dismiss option for dapp connections ([#3836](https://github.com/Agoric/agoric-sdk/issues/3836)) ([f193fa2](https://github.com/Agoric/agoric-sdk/commit/f193fa225c6e2e88b067d43e22b1fcc1230ebb98))
+* **wallet:** Don't show autodeposit payments in requests, smooth out rendering animations ([#3832](https://github.com/Agoric/agoric-sdk/issues/3832)) ([b03c7a5](https://github.com/Agoric/agoric-sdk/commit/b03c7a5d44093c356d29e807a93041a760b1a029))
+* **wallet:** Eagerly show pending state for approved offers ([#3846](https://github.com/Agoric/agoric-sdk/issues/3846)) ([20abcbe](https://github.com/Agoric/agoric-sdk/commit/20abcbed5bfab6c5a8ad19462b9b726fd2d6490b))
+* **wallet:** Fix icons and list rendering bugs ([#3827](https://github.com/Agoric/agoric-sdk/issues/3827)) ([02b7554](https://github.com/Agoric/agoric-sdk/commit/02b7554076bddbe878756ac44af736c4b5d282c2))
```
Comment (dckc) [inline CHANGELOG.md:17]: "Fix ... bugs" looks more like a `fix` than a `feat`

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,38 @@
...
+* **SwingSet:** improve slogging during replay ([d6e64da](https://github.com/Agoric/agoric-sdk/commit/d6e64daf4d8240ad51eafb917e6f4590c8caebfb)), closes [#3428](https://github.com/Agoric/agoric-sdk/issues/3428)
+* **timer:** remove deprecated `createRepeater` ([b45c66d](https://github.com/Agoric/agoric-sdk/commit/b45c66d6d5aadcd91bd2e50d31104bce8d4d78f6))
+* **xsnap:** format objects nicely in console using SES assert.quote ([#3856](https://github.com/Agoric/agoric-sdk/issues/3856)) ([a3306d0](https://github.com/Agoric/agoric-sdk/commit/a3306d01d8e87c4bc7483a61e42cc30b006feb81)), closes [#3844](https://github.com/Agoric/agoric-sdk/issues/3844)
+* fix bug which breaks rights conservation and offer safety guarantees ([#3858](https://github.com/Agoric/agoric-sdk/issues/3858)) ([b67bfcb](https://github.com/Agoric/agoric-sdk/commit/b67bfcb9051cdcf780aff1a10653635448b21eae))
+* skip refill meter test ([#3849](https://github.com/Agoric/agoric-sdk/issues/3849)) ([90a456f](https://github.com/Agoric/agoric-sdk/commit/90a456f78f918ad01924da4b131f5a272d03624b))
```
Comment (dckc) [inline CHANGELOG.md:34]: Changes to tests don't seem like they could be user-visible `fix`es.

File: packages/sparse-ints/package.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@agoric/sparse-ints",
-  "version": "0.1.22",
+  "version": "0.1.23",
   "description": "Share code across vat boundaries",
```
Comment (dckc) [inline packages/sparse-ints/package.json:4]: sparse-ints description looks like copy-pasta

File: packages/tame-metering/CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+### [2.0.13](https://github.com/Agoric/agoric-sdk/compare/@agoric/tame-metering@2.0.12...@agoric/tame-metering@2.0.13) (2021-09-23)
```
Comment (dckc) [inline packages/tame-metering/CHANGELOG.md:6]: we don't need tame-metering any more, do we?

**Review by dckc — APPROVED** [review]
ship it!

all the comments here are just thinking out loud

File: CHANGELOG.md (hunk 6 above)
Comment (michaelfig) [reply CHANGELOG.md:34]: Agreed, should have been `test(xxx): ...`.

File: packages/sparse-ints/package.json (hunk 7 above)
Comment (michaelfig) [reply packages/sparse-ints/package.json:4]: Hmm.  I'll probably drive-by fix it next time I think of it.

File: packages/tame-metering/CHANGELOG.md (hunk 8 above)
Comment (michaelfig) [reply packages/tame-metering/CHANGELOG.md:6]: Right.

**Pull request merged**